### PR TITLE
feat(workflows): add github workflows, codeowners

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

**Review by tedk13 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.github/** @ritterim/GitGuardians
```
Comment (mrtrimble) [inline .github/CODEOWNERS:3]: is this OK? it says `This CODEOWNERS file contains errors`

image

Comment (hougasian) [reply .github/CODEOWNERS:3]: should be fine. copied from [skellyCSS](https://github.com/ritterim/skellyCSS/blob/main/.github/CODEOWNERS)

**Review by tedk13 — APPROVED** [review]

**Review by tedk13 — APPROVED** [review]
Just Approve it, @mrtrimble !

**Review by mrtrimble — APPROVED** [review]

**Comment by mrtrimble** [comment]
I'm approving and merging this, but not because Ted told me to

**Pull request merged**